transfermethodui/src/main/java/com/hyperwallet/android/ui/transfermethod/view/widget/TextWidget.java

```diff
@@ -75,6 +75,7 @@ public void onFocusChange(View v, boolean hasFocus) {
             editText.setText(mDefaultValue);
             editText.setInputType(InputType.TYPE_CLASS_TEXT);
             editText.setOnKeyListener(new DefaultKeyListener(mDefaultFocusView, editText));
+            editText.addTextChangedListener(new TextChangeListener());
```

Comment:
There is already a TextChangedListener for TextWidget. Why not use the existing one?

Reply:
https://developer.android.com/reference/android/view/View.OnKeyListener
OnKeyListener not triggered in some keyboard, google recommended to use textchange listener for text changes.

Reply:
fixed